lib/index.js

```diff
@@ -1,6 +1,21 @@
 'use strict';
 
-var rollup = require('rollup');
+const fs = require('fs');
+const rollup = require('rollup');
+const debounce = require('debounce');
+
+const dependencies = new Map()
+const changedParents = new Set()
+const WAIT = 25
+
+const touchParents = debounce(() => {
```

Comment:
I used the top-rated answer here http://stackoverflow.com/questions/28388885 as a guide.

Unless I misunderstood, it looks like for-of, Map/Set, and const are supported in Node 0.12.  So I removed the arrow functions and destructuring assignment.

ES6 is preferable not for ease, but for beauty. :smile: